### PR TITLE
Presentation detail link proposal

### PR DESCRIPTION
**Pull request by miurahr**
Implemented #91

### CODE REVIEW

File: symposion/proposals/models.py [hunk 1]
```diff
@@ -147,6 +149,12 @@ def notification_email_context(self):
             "kind": self.kind.name,
         }
 
+    def save(self, *args, **kwargs):
```
Comment (paltman) [inline symposion/proposals/models.py]: What about #110?

**Comment by paltman** [comment]
@jtauber was there a specific reason for the copying over of these fields?  Could they ever change on one but not the other?  If not, I think this is a decent change.

**Comment by miurahr** [comment]
@jtauber I've tested this code on my project and it does not use `review` app.
I'm afraid whether review app and this modification work well or not.

File: symposion/proposals/models.py (hunk 1 above)
Comment (miurahr) [inline symposion/proposals/models.py]: #110 is for a slot name.  here is for a proposal with same way.

**Comment by miurahr** [comment]
rebased.

**Comment by miurahr** [comment]
This PR make a bug to reverse-reference from Speaker to Presentation. Because it drop speaker field from Presentation and add a function to refer Proposal.speaker. This cause a  problem.

**Comment by miurahr** [comment]
I'd like to mark it Invalid and close.

**Pull request closed without merging**